azure-pipelines-clean-up.yml

```diff
@@ -0,0 +1,44 @@
+stages:
```

Comment:
I think this can just be named clean-up-branches.yml or something. azure-pipelines.yml is just the default name that azure devops looks for when you're trying to create a new pipeline, so if we're not calling it exactly that, might as well just call it for what it does? current name kind of implies you're cleaning up azure pipelines

Reply:
Renamed the yaml to something that won't get your hopes up on what it does. :)